README.md

```diff
@@ -1,2 +1,43 @@
 # Image-Filter-Application
-### `An Application to apply different Filters to different BMPs Images`
\ No newline at end of file
+### `An Application to apply different Filters to different BMPs Images`
+## Table of Contents
+
+1) [An overview](#project)
+2) [Building Image-Filter-Application](#building)
+3) [Running Image-Filter-Application](#running)
+
+## Project
+
+This Project is Associated with Department of Integrated MCA Utkal Uniersity, Bhubaneshwar, Odisha.
+## Building
+
```

Comment:
Create a separate section or division from line 14 to 16 and label it as dependencies and prerequisite